frontends/ol-components/src/components/RoutedDrawer/RoutedDrawer.tsx

```diff
@@ -67,7 +69,8 @@ const RoutedDrawer = <K extends string, R extends K = K>(
       })
       return newSearchParams
     })
-  }, [setSearchParams, params])
+    document.location.hash = hash.substring(1)
```

Comment:
In general, in a React app, we shouldn't touch `documentation.location` (or `window.location`):
- Changes to those properties won't cause React to re-render, which could be lead to bugs
- Will usually cause a full page reload, which we don't want. *This appears not to be the case when only changing the hash, but the first point is still relevant.*

Instead use `useNavigate`. Probably you'll want something like
```ts
  const removeUrlParams = useCallback(() => {
    const newSearchParams = new URLSearchParams(location.search)
    params.forEach((param) => {
      newSearchParams.delete(param)
    })
    navigate({
      ...location,
      search: newSearchParams.toString(),
    })
  }, [navigate, location])
```